integration/integration_test.go

```diff
@@ -7223,6 +7226,160 @@ func testSFTP(t *testing.T, suite *integrationTestSuite) {
 	require.Equal(t, testFilePath, sftpEvent.GetString(events.SFTPPath))
 }
 
+func testAgentlessConnection(t *testing.T, suite *integrationTestSuite) {
+	// create Teleport instance
+	teleInst := suite.newTeleport(t, nil, true)
+	t.Cleanup(func() {
+		teleInst.StopAll()
```

Comment:
```suggestion
		_ = teleInst.StopAll()
```

so it's clear we are swallowing the error.

I've other folks do this too:

```suggestion
		require.NoError(teleInst.StopAll())
```

(No strong opinions on my part, as long as we pick one.)

Reply:
Done